### ISSUE TITLE
image in svg not loaded

### ISSUE THREAD

**Issue by aboutlo** [fuzhenn/node-svg2img#1]
I trying to load an image in my svg:

```
<image width="24" height="24" x="6" y="5" href="https://s3-eu-west-1.amazonaws.com/assets.theoak.co/categories/food_and_dining.png"></image>
```
First I got:
`ImageClass` isn't a constructor

then I saw I can pass an ImageClass constructor in the options so that I tried CanvasImage. 
The error is fixed but nothing hasn't been loaded. 

Is ImageClass constructor supported? If so Can you show an example?

Cheers

**Comment by fuzhenn** [comment]
Hi, @aboutlo

Yeah, it is indeed a bug, I fixed it in the referenced commit and published it to npm as 0.2.4.

But here is something you may need to know:

Because canvg is originally designed in browser not for node, it cannot load remote images in SVG  properly in node environment.
You need to load the images and convert to base64 string before passing to ``svg2img``.

e.g.
```javascript
var fs = require('fs');
var util = require('util');
var Image64 = require('node-base64-image');
var svg2img = require('./');

var imageUrl = 'https://www.baidu.com/img/bd_logo1.png';
// load the image, and get its base64 string
Image64.encode(imageUrl, {}, function (err, base64) {
    // convert remote image url to base64
    var svgString = util.format('<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="540" height="258" ' +
'viewBox="0 0 540 258"><image width="540" height="258" x="6" y="5" href="%s"></image></svg>', 'data:image/png;base64,' + base64.toString('base64'));
    svg2img(svgString, function(error, buffer) {
        fs.writeFileSync('foo.png', buffer);
    });
});
```

I also add an unit test, you can check it out here. https://github.com/fuzhenn/node-svg2img/blob/master/test/specs.js#L76

Hope it helps 🐱

**Comment by aboutlo** [comment]
Thank you @fuzhenn I appreciate you did SOOO fast :). 
I see the issue, I will generate the base64 encoding before.
